moderntimeline.dtx

```diff
@@ -487,6 +558,10 @@
    \def\tl@firstyear{#1}
    \def\tl@lastyear{#2}
    \pgfmathsetmacro\tl@yearrange{\tl@lastyear-\tl@firstyear}
+   \pgfmathsetmacro\tl@totalyears{\tl@lastyear-\tl@firstyear}%
```

Comment:
As stated in my earlier comment, this is a verbatim copy of `\tl@yearrange` above.